examples/anthos-service-mesh-multicluster/infrastructure/clusters.tf

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# create the clusters
+# https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-shared-vpc#creating_a_cluster_in_your_first_service_project
+resource "google_container_cluster" "cluster3" {
```

Comment:
nit: you could look at using the GKE module which would reduce the config required for this setup. [Example](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/d473b14adecd42c12e2024fed53f1bfe70b13602/examples/simple_zonal_with_asm/main.tf#L35)

Reply:
Excellent suggestion. This was a pretty awesome fix